nbodykit/algorithms/fftpower.py

```diff
@@ -295,7 +295,7 @@ def run(self):
             kedges, kcoords = _find_unique_edges(y3d.x, 2 * numpy.pi / y3d.BoxSize, kmax, y3d.pm.comm)
 
         # project on to the desired basis
-        muedges = numpy.linspace(0, 1, self.attrs['Nmu']+1, endpoint=True)
+        muedges = numpy.linspace(-1, 1, self.attrs['Nmu']+1, endpoint=True)
```

Comment:
This may break some tests. Did you run runtest.py --single  locally?

Reply:
I'm having some trouble running runtest.py --single locally (seems to be trouble with MPI). Can I just copy the directory to nersc and run runtest.py there?